rosdep/base.yaml

```diff
@@ -3304,6 +3312,7 @@ libssl-dev:
   alpine: [libressl-dev]
   arch: [openssl]
   debian: [libssl-dev]
+  freebsd: [openssl]
```

Comment:
This is out of alphabetical order which is failing CI

Reply:
Doh! Pardon my messing up on the ordering.  I have corrected that in the next commit.

Also, thank you for the tip about Superflore! That will definitely get a more complete package listing than my ad-hoc method thus far. I'll take a look.

I presented ROS at a BSD conference over the weekend and it has generated some enthusiasm for getting things into the FreeBSD ports tree. So, it should lead to others being able to use FreeBSD and ROS without extensive hacking.